openstf/model/basecase.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from datetime import datetime, timedelta, timezone
+import pytz
+import numpy as np
+import pandas as pd
+
+
+from sklearn.base import BaseEstimator, RegressorMixin
+
+MINIMAL_RESOLUTION: int = 15  # Used for validating the forecast input
+
+
+class BaseCaseModel(BaseEstimator, RegressorMixin):
+    def predict(self, forecsst_input_data: pd.DataFrame) -> pd.DataFrame:
+        """Predict using the basecase method
+
+        Args:
+            forecsst_input_data (pd.DataFrame): Forecast input dataframe
+
+        Returns: pd.DataFrame: Basecase forecast
+
+        """
+        return self.make_basecase_forecast(forecsst_input_data)
+
+    @staticmethod
+    def make_basecase_forecast(
+        forecast_input_data: pd.DataFrame, overwrite_delay_hours: int = 48
+    ) -> pd.DataFrame:
+        """Make a basecase forecast. THe idea of the basecase forecast is that if all else fails, this forecasts is
+            still available.
+            Basecase example: the load of last week.
+        Args:
+            forecast_input_data (pd.DataFrame): Forecast input dataframe
+            overwrite_delay_hours (float): times before this in the future are not
+                forecasted
+
+        Raises:
+            ValueError: In case T-7d or T-14d is not present a ValueError is raised.
+            Also in the case the start of the forecast is before the horizon of the regular forecast,
+             a value error is raised
+        Returns:
+            pd.DataFrame: Basecase forecast
+
```

Comment:
Good docstring, very clear!
Minor remark: under 'raises', it would be more clear if it was like this:


```
Raises:
    ValueError: if columns T-7d or T-14d is not present
    ValueError: If the start of the forecast is before the horizon of the regular forecast
```